sdk/appplatform/Microsoft.Azure.Management.AppPlatform/src/Generated/Models/AppResourceProperties.cs

```diff
@@ -39,18 +39,21 @@ public AppResourceProperties()
         /// the App</param>
         /// <param name="fqdn">Fully qualified dns Name.</param>
         /// <param name="httpsOnly">Indicate if only https is allowed.</param>
+        /// <param name="enableEndToEndTLS">Indicate if end to end TLS is
+        /// enabled.</param>
         /// <param name="createdTime">Date time when the resource is
         /// created</param>
         /// <param name="temporaryDisk">Temporary disk settings</param>
         /// <param name="persistentDisk">Persistent disk settings</param>
-        public AppResourceProperties(bool? publicProperty = default(bool?), string url = default(string), string provisioningState = default(string), string activeDeploymentName = default(string), string fqdn = default(string), bool? httpsOnly = default(bool?), System.DateTime? createdTime = default(System.DateTime?), TemporaryDisk temporaryDisk = default(TemporaryDisk), PersistentDisk persistentDisk = default(PersistentDisk))
+        public AppResourceProperties(bool? publicProperty = default(bool?), string url = default(string), string provisioningState = default(string), string activeDeploymentName = default(string), string fqdn = default(string), bool? httpsOnly = default(bool?), bool? enableEndToEndTLS = default(bool?), System.DateTime? createdTime = default(System.DateTime?), TemporaryDisk temporaryDisk = default(TemporaryDisk), PersistentDisk persistentDisk = default(PersistentDisk))
```

Comment:
This would be a breaking change due to the field being added NOT at the end of the properties list in the swagger. Can you submit a fix on the swagger and regenerate so we can avoid this breaking change in .NET?

Reply:
new field should be at the end of the properties list, so it won't break the .net SDK, is that right?
sorry I did not know this.
Is this a .net specific requirement? Please share me more background details

I will submit the new swagger file change
Also regenerate the .net sdk files

Reply:
@allenjzhang updated, please review